PrivateCollections/Germany/CHJN007.xml

```diff
@@ -0,0 +1,1175 @@
+                   <dim unit="leaf">8</dim>
+                   <locus from="52" to="59"/>
+              </item>
+              <item xml:id="q9" n="8">
+                 <dim unit="leaf">8</dim>
+                 <locus from="60" to="67"/>
+            </item>
+            <item xml:id="q10" n="9">
+               <dim unit="leaf">8</dim>
+               <locus from="68" to="75"/>
+             </item>
+             <item xml:id="q11" n="10">
+                <dim unit="leaf">8</dim>
+                <locus from="76" to="83"/>
+              </item>
+              <item xml:id="q12" n="11">
+                 <dim unit="leaf">10</dim>
+                 <locus from="84" to="93"/>
+               </item>
+               <item xml:id="q13" n="12">
+                 <dim unit="leaf">8</dim>
+                 <locus from="94" to="101"></locus>
+               </item>
+               <item xml:id="q14" n="13">
+                 <dim unit="leaf">8</dim>
+                 <locus from="102" to="109"></locus>
+               </item>
+               <item xml:id="q15" n="14">
+                 <dim unit="leaf">8</dim>
+                 <locus from="110" to="118"></locus>
+               </item>
+               <item xml:id="q16" n="15">
+                 <dim unit="leaf">8</dim>
+                 <locus from="118" to="125"></locus>
+               </item>
+               <item xml:id="q17" n="16">
+                 <dim unit="leaf">8</dim>
+                 <locus from="126" to="133"></locus>
+               </item>
+               <item xml:id="q18" n="17">
+                 <dim unit="leaf">8</dim>
+                 <locus from="134" to="141"></locus>
+               </item>
+               <item xml:id="q19" n="18">
+                 <dim unit="leaf">8</dim>
+                 <locus from="142" to="149"></locus>
+               </item>
+               <item xml:id="q20" n="19">
+                 <dim unit="leaf">8</dim>
+                 <locus from="150" to="157"></locus>
+               </item>
+               <item xml:id="q21" n="20">
+                 <dim unit="leaf">5</dim>
+                 <locus from="158" to="162"></locus>
+                 158, stub after 162
+               </item>
+               <item xml:id="q22" n="21">
+                 <dim unit="leaf">2</dim>
+                 <locus from="163" to="164"></locus>
+               </item>
+                            </list>
+                          </collation>
+                          <condition key="good">Stains of dirt throughout the manuscript. The
+                            first and the last leaves are damaged by insects.
+                            <locus target="#86 #106 #112 #113 #114 #115 #116 #119 #121 #122
+                              #125 #126 #127 #132 #133"></locus> are torn or damaged and repaired.
+                              The right lower corner of <locus target="#158"></locus> is torn down.
+                              The last quire is incomplete.
```

Comment:
I don't exactly understand what is meant by "the last quire is incomplete". Is it a bifolium, but you can see that folia are missing in the middle?

Reply:
You are right, the text is incomplete, but the quire is complete.

Reply:
Aha, now I understand!